meta-efi-secure-boot/recipes-base/packagegroups/packagegroup-efi-secure-boot.bb

```diff
@@ -8,12 +8,15 @@ S = "${WORKDIR}"
 
 ALLOW_EMPTY_${PN} = "1"
 
+
+SELOADER_PKG = "${@'seloader' if d.getVar('UEFI_SELOADER', True) == '1' else ''}"
+
```

Comment:
Extra blank lines added.

Reply:
Perhaps I missed labeled a patch.  I don't believe at the current time there is any reason to push the strict verification for grub, so I deemed it in appropriate for upstream work.

Reply:
Alright.  But the end result is:
ALLOW_EMPTY_${PN} = "1"


SELOADER_PKG = ...
And the double blank is what I was worried about :)

Reply:
I was actually commenting on the top level question.  The white space is fixed in v2.